test/extern/fnPtrs/sugarFcfTypes2.future

```diff
@@ -1,4 +1,4 @@
-error message: Need to sugar first-class function (fcf) types
-
-We shouldn't be printing out internal fcf type names when we
-generate errors involving first-class functions.
+sugarFcfTypes2.chpl:13: error: unresolved call 'barIntInt(proc(x: int): int)'
```

Comment:
My understanding was that futures should describe the issue in human-readable format (that's what's the "before" is doing in this file). However, it seems like you've put in the "desired error message format", without any kind of description. Doesn't this stuff go into `sugarFcfTypes2.good`?

Reply:
Hmmm, nope, somehow I think I cp'd to the wrong file name, ty!

Reply:
I promoted these futures.